### PR TITLE
Update README.md

### LINKED ISSUE #41: ReadMe Updates
Need to update the readMe with relevant information

### LINKED ISSUE #67: Add new deployment link to README
**Description of what needs refactoring**
A brief description of the code that needs refactoring and the reason for refactoring
- readme currently rickrolls unsuspecting patrons

**Additional Context** 
File/function name or additional reasoning for the changes
- `README.md`

**Screenshot of Issue**
Include a screenshot if it would help describe the issue

### LINKED ISSUE #68: Fix deployment link in readme
**Description of what needs refactoring**
A brief description of the code that needs refactoring and the reason for refactoring
- readme currently rickrolls unsuspecting patrons

**Additional Context** 
File/function name or additional reasoning for the changes
- `README.md`

**Screenshot of Issue**
Include a screenshot if it would help describe the issue

### LINKED ISSUE #69: Fix images in README
**Description of what needs refactoring**
A brief description of the code that needs refactoring and the reason for refactoring
- readme currently has some rather lewd kitchen behavior displayed 

**Additional Context** 
File/function name or additional reasoning for the changes
- `readme.md`

**Screenshot of Issue**
Include a screenshot if it would help describe the issue

### PR DESCRIPTION
**Pull request by richardltyler**
## *Purpose:*
*What does this merge do? Check all that apply.*
- [x] Adds a new feature
- [ ] Improves an existing feature
- [ ] Fixes a bug
- [ ] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*
*Briefly, but clearly describe what you did, and explain any new code.*
- adds README

## *Instructions for reviewer:*
*Where should the reviewer start?*
*How should this be manually tested?*
*Any background context you want to provide?*
*What are the relevant tickets?*
closes #41 

## *Issues:*
*Are there any concerns, issues, or bugs in this branch? If so describe them.*

## Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have used tests to prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,105 +1,64 @@
...
 
-## Test Files Organization
-
-Similar to feature code, your test code needs to be put in a specific place for it to run successfully.
-
-**Put all of your test files in the `test` directory.** As a convention, all test filenames should end with `-test.js`. For instance: `box-test.js`.
-
-## Running Your Tests
+## Introduction
+The primary goal of [Whats-Cookin](https://github.com/alia-peterson/whats-cooking) is to help users plan their meals. They can search through a log of recipes, save their favorite recipes, and keep track of their pantry. The specifications can be found [here](https://frontend.turing.io/projects/whats-cookin.html) and [here](https://frontend.turing.io/projects/module-2/refactor-tractor-wc.html)
```
Comment (tashiad) [inline README.md]: Add a period at the end of this paragraph.

File: README.md [hunk 2]
```diff
@@ -1,105 +1,64 @@
...
 
-Run your test suite using the command:
+## Motivation
+The motivation behind this project's creation was to get experience working in an existing code base. It also allowed the opportunity to ractice using `Fetch` API and updating code to incorporate more modern functionality.
```
Comment (tashiad) [inline README.md]: - remove "creation" so it just says "The motivation behind this project was..."
- "ractice" is missing a "p" at the beginning 🙃

File: README.md [hunk 3]
```diff
@@ -1,105 +1,64 @@
...
 
-Your linter will look at the JavaScript files you have within the `src` directory and the `test` directory. 
+#### Search Ideas
+You'll find all of the recipes you need here, but that can be a lot to handle, so we got a search bar, baby! Just type in a recipe name or ingredient, and we'll find everything that suits your needs.  
+<p align = "center">
+<img src="https://lwgsummerland.files.wordpress.com/2013/06/sexyvegeman.jpg">
+</p>
+    <details>
+      <summary>Under the Hood</summary>
+      Some Stuff
```
Comment (tashiad) [inline README.md]: This is cool! Never seen this in markdown before.

File: README.md [hunk 4]
```diff
@@ -1,105 +1,64 @@
...
 
-## Deploying to GitHub Pages
+#### Continuous Improvement/Future Improvements
 
-_If you are finished with the functionality and testing of your project_, then you can consider deploying your project to the web! This way anyone can play it without cloning down your repo.
+---
 
-[GitHub Pages](https://pages.github.com/) is a great way to deploy your project to the web. Don't worry about this until your project is free of bugs and well tested!
+## Technologies
+JS, Fetch/Async, HTML, & CSS/SCSS, Normalize, Webpack
```
Comment (tashiad) [inline README.md]: - Remove &
- Could maybe include JSON, Mocha, and Chai?

File: README.md [hunk 5]
```diff
@@ -1,105 +1,64 @@
...
+
+## Deployment
+This app requires a local server to be running independent of GH. Clone down [this repo](https://github.com/turingschool-examples/whats-cookin-api) and follow the instructions in the README. Once the server is running on your local machine, the site can be seen in all of it's glory here: 
+
+[Deployment Link](https://youtu.be/dQw4w9WgXcQ)
```
Comment (tashiad) [inline README.md]: This is the second time this week that I've been rick rolled 😂

**Review by tashiad — APPROVED** [review]
Love this format! Just a few minor edits/typos.

File: README.md (hunk 4 above)
Comment (richardltyler) [reply README.md]: good call

File: README.md [hunk 6]
```diff
@@ -1,105 +1,130 @@
...
+<p align = "center">
+<img src="https://media.giphy.com/media/8J8SBaqrNxbPz8jla8/giphy.gif">
+</p>
+    <details>
+      <summary>Under the Hood</summary>
+      When the user selects the apple icon, the card element's class is updated. Then, when the user clicks "My Recipes", the main section's class is updated to only display the cards with the favorited class. 
+    </details>
+   
+#### Pantry
+The user can also keep track of their pantry with this app. In the top, right corner, the user can select the "My Pantry" button to see all of the ingredients that they have in their cupboard to helo decide what recipes they want to make.
```
Comment (tashiad) [inline README.md]: "helo" typo! should be help.

File: README.md [hunk 7]
```diff
@@ -1,105 +1,130 @@
...
-
-## Webpack?
-
-If you look in the `package.json` file, you'll see one of the library dependencies called `webpack`. If you're interested in learning more about what Webpack is and how it works behind the scenes, take a look through the [Webpack configuration documentation](https://webpack.js.org/concepts/).
-
-## Deploying to GitHub Pages
+---
 
-_If you are finished with the functionality and testing of your project_, then you can consider deploying your project to the web! This way anyone can play it without cloning down your repo.
+## Features
```
Comment (tashiad) [inline README.md:20]: Love this section! The under the hood stuff is so great.

File: README.md [hunk 8]
```diff
@@ -1,105 +1,130 @@
...
+    <br>
+    If the user has the required ingredients, they can click the "Cook Recipe" button and the User's pantry will be uodated using a a Fetch POST to remove the ingredients. If the user does not have the necessary ingredients, they can click the "Add Items to Pantry" button and the User's pantry will be updated using a Fetch POST to add the ingredients.
+    <br>
+    <br>
+    When the user decides to cook a recipe, the Recipe is added to the User's cookedRecipes property. If the user has cooked the recipe before and it already exists in that property, then the Recipe is not added, but the date is updated. 
+    </details>
+
+#### Continuous Improvement/Future Improvements
+- Check for duplicate ingredients in the API before being added to the classes
+- Style more responsively
```
Comment (tashiad) [inline README.md]: Lets be a little more specific:
"Improve mobile responsive design so that the Recipe Type filter sidebar is shown/hidden on click."

**Review by tashiad — APPROVED** [review]
Approved--with the changes I outlined in the comments!

File: README.md [hunk 9]
```diff
@@ -1,105 +1,130 @@
...
+
+#### Search Recipes
+You'll find all of the recipes you need here, but that can be a lot to handle, so we got a search bar, baby! Just type in a recipe name or ingredient, and we'll find everything that suits your needs.  
+<p align = "center">
+<img src="https://media.giphy.com/media/94qT4z80QExCPprrmX/giphy.gif">
+</p>
+    <details>
+      <summary>Under the Hood</summary>
+      Everytime a user types into the search bar, the recipes are filtered by their name for recipes that do not meet the search criteria. Then those unmatching recipes are filtered for recipes whose ingredients do not include the search criteria. Then the recipes whose names and ingredients do not meet the search criteria are hidden from the DOM
+    </details>
```
Comment (alia-peterson) [inline README.md:48]: I feel like this reads a little weird. Maybe update to something like this:
When the user starts to type into the search bar, the recipes are filtered first by name and then by ingredient. The recipes that do not match the search item are then hidden from view.

**Review by alia-peterson — APPROVED** [review]

**Review by alia-peterson — APPROVED** [review]

**Pull request merged**